ministryofjustice/analytics-platform-control-panel
# Embedding QS into Control Panel

## :memo: Summary
This PR resolves ...


This PR adds QS into the control panel, similar to the AP UI.

Merging this PR will have the following side-effects:
- Quicksight will now be in the control panel

## :mag: What should the reviewer concentrate on?
- Feedback on specific parts of the code
- Check side effects, if any

## :technologist: How should the reviewer test these changes?
- Pull down branch
- ensure you are a superuser locally
- Add the following to your .env file:
   - export QUICKSIGHT_ACCOUNT_ID="381491960855"
   - export QUICKSIGHT_ACCOUNT_REGION="eu-west-2"
   - export QUICKSIGHT_DOMAINS="http://localhost:8000"
   - export QUICKSIGHT_ASSUMED_ROLE="arn:aws:iam::381491960855:role/analytical-platform-control-panel"
- Run control panel locally
- Navigate to quicksight page
- See quicksight in all its glory

## :books: Documentation status
